src/file.ts

```diff
@@ -93,3 +107,47 @@ export async function getEntryData(
   }
   return { data: entry.data };
 }
+
+// ===============
+// Encrypted Files
+// ===============
+
+/**
+ * Gets Encrypted JSON set with MySky at the given data path for the given
+ * public user ID.
+ *
+ * @param this - SkynetClient
+ * @param userID - The MySky public user ID.
+ * @param pathSeed - The share-able secret path seed.
+ * @param [customOptions] - Additional settings that can optionally be set.
+ * @returns - An object containing the decrypted json data.
+ */
+export async function getEncryptedJSON(
+  this: SkynetClient,
+  userID: string,
+  pathSeed: string,
+  // TODO: Take a new options type?
+  customOptions?: CustomGetJSONOptions
+): Promise<EncryptedJSONResponse> {
+  validateString("userID", userID, "parameter");
+  validateStringLen("pathSeed", pathSeed, "parameter", 64);
+  validateOptionalObject("customOptions", customOptions, "parameter", defaultGetJSONOptions);
+
+  const opts = {
```

Comment:
Couldn't leave without at least one `nit` right. I think the following reads a bit better. I decided to comment on it because I also wanted to ask why `JsonFullData` defines `_data` and not just `data`, what's the idea with the underscores. 

Maybe we should `getRawBytes` to simply `getData`? I was expecting `getRawBytes` to return a uint8array when I read it the first time.

```
const opts = {
    ...defaultGetJSONOptions,
    ...this.customOptions,
    ...customOptions,
    hashedDataKeyHex: true
}

// fetch the raw encrypted JSON data
const dataKey = deriveEncryptedFileTweak(pathSeed);
const rawBytes = await this.db.getRawBytes(userID, dataKey, opts);
if (rawBytes.data === null) {
    return { data: null };
}

// decrypt it
const key = deriveEncryptedFileKeyEntropy(pathSeed);
const { _data: data } = decryptJSONFile(rawBytes.data, key);
return { data };
```

Reply:
> why JsonFullData defines _data and not just data, what's the idea with the underscores.

To emphasize that they are hidden fields that the user should never see.

> Maybe we should getRawBytes to simply getData? I was expecting getRawBytes to return a uint8array when I read it the first time.

 It does. 😛